examples/model_loader/source/ui_helper.hpp

```diff
@@ -0,0 +1,181 @@
+#include <gvk.hpp>
+#include <imgui.h>
+#include <any>
+
+/** ImGui for some reason saves its arithmatic types into some enum - determine the type
+*/
+class imgui_data_type_determinator
```

Comment:
These ui helpers are pretty cool. Maybe we should consider adding them to the framework as utilities and not just have them lying around in a sample?!

Reply:
Yes, I wanted to suggest that. however they are not mature enough yet and need an extra issue number i think so someone can make them a bit cooler.

Reply:
Most importantly, some of them are not general enough.